libs/vre/shared/app-analytics/src/lib/grafana-faro.service.ts

```diff
@@ -9,7 +9,10 @@ export class GrafanaFaroService {
   constructor(private readonly _appConfig: AppConfigService) {}
 
   setup() {
-    if (this._appConfig.dspInstrumentationConfig.environment === 'local-dev') {
+    if (
+      this._appConfig.dspInstrumentationConfig.environment === 'local-dev' ||
+      this._appConfig.dspInstrumentationConfig.environment === 'dev-server'
```

Comment:
['local-dev', 'dev-server'].includes(this._appConfig.dspInstrumentationConfig.environment) seems better